src/common/components/cookie-notification/index.js

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from 'react';
+
+// importing styles directly from cookie-policy module
+import styles from '../../../../node_modules/cookie-policy/build/css/cookie-policy.css';
```

Comment:
Doesn't the sass command have an `includePath` to node_modules?

Reply:
We use CSS Modules in BSI, not sass directly. So at this point we can't really use cookie-policy sass here.
Fortunately it's convenient enough to import built CSS as CSS Module which works just fine.